src/views/notifications/OrderStatusNotification.tsx

```diff
@@ -80,7 +79,7 @@ export const OrderStatusNotification = ({
         );
       }
       break;
-    default: // OrderSubmissionStatuses.Submitted
+    case PlaceOrderStatuses.Submitted:
```

Comment:
eh I prefer using all the explicit cases if `default` isn't actually necessary (so that if anyone ever adds a case it's very obvious that they need to consider it here)